src/cli/graph.js

```diff
@@ -126,15 +160,81 @@ export function _hackyIdentityNameReferenceDetector(
 export const graphHelp: Command = async (args, std) => {
   std.out(
     dedent`\
-      usage: sourcecred graph
+      usage: sourcecred graph [-d]
 
       Generate a graph from cached plugin data
 
       Either 'sourcecred load' must immediately precede this command or
       a cache directory must exist for all plugins specified in sourcecred.json
+
+      If the -d flag is provided, it will also compute and output
+      any resulting graph changes since the last generated graph.
       `.trimRight()
   );
   return 0;
 };
 
+function computeAndLogDiff(
+  oldWeightedGraph: WeightedGraph,
+  newWeightedGraph: WeightedGraph,
+  pluginName: string
+) {
+  const graphDiff = compareGraphs(
+    oldWeightedGraph.graph,
+    newWeightedGraph.graph
+  );
+  const horizontalRule =
+    "=============================================================";
+  if (graphDiff.uniqueNodesInFirst.length > 0) {
+    console.log(
+      `${horizontalRule}\n  ${pluginName} - Unique Nodes in the Old graph\n${horizontalRule}`
+    );
+    console.log(graphDiff.uniqueNodesInFirst);
```

Comment:
Throughout this file: Don't directly console.log nodes or edges, because the addreses contain NUL bytes (the `\x00`) which makes them hard to read. Instead, use the `nodeToString` and `edgeToString` functions:

https://github.com/sourcecred/sourcecred/blob/ed3de0d9e5cc6fe991d5b2e197c9e3728798e4d8/src/core/graph.js#L1158-L1178

(Note--the `nodeToString` function needs to be updated to include the timestamp and description, looks like I forgot to update it when I added those fields. Recommend doing that in a small separate PR.)

Reply:
I'll change this in my second pass when adding weights.